tests/test_transcript.py

```diff
@@ -317,3 +306,30 @@ def test_regex_transcript(request, capsys):
     else:
         assert err == ''
         assert out == ''
+
+
+@pytest.mark.parametrize('expected, transformed', [
```

Comment:
I am not a regex guy and am fundamentally unqualified to review this code.  But you really seem like you know what you are doing, so thanks for the good work.  Your code modifications seem to have excellent test coverage.